src/Nox.Generator/Application/Dto/EntityDto.template.cs

```diff
@@ -11,6 +11,7 @@
 using Nox.Domain;
 using {{codeGeneratorState.DataTransferObjectsNameSpace}};
 using {{codeGeneratorState.DomainNameSpace}};
+using Microsoft.OData.ModelBuilder;
```

Comment:
this using is strange? EntityDto should not dependn on Odata?

Reply:
Errors I receive without it:
Nox.Generator\src\SampleWebApp\Nox.Generator\Nox.Generator.NoxCodeGenerator\Dto.CountryDto.g.cs(110,3,110,13): error CS0246: The type or namespace name 'AutoExpandAttribute' could not be found (are you missing a using directive or an assembly reference?)
Nox.Generator\src\SampleWebApp\Nox.Generator\Nox.Generator.NoxCodeGenerator\Dto.CountryDto.g.cs(110,3,110,13): error CS0246: The type or namespace name 'AutoExpand' could not be found (are you missing a using directive or an assembly reference?)

Reply:
Let's discuss, maybe we don't need it